outputs.tf

```diff
@@ -3,10 +3,20 @@
 # ------------------------------------------------------------------------------
 output "arn" {
   description = "The ARN of the certificate that is being validated."
-  value       = "${element(concat(aws_acm_certificate_validation.main.*.certificate_arn, list("")), 0)}"
+  value = element(
+    concat(aws_acm_certificate_validation.main.*.certificate_arn, [""]),
+    0,
+  )
 }
 
 output "wildcard_arn" {
   description = "The ARN of the wildcard certificate that is being validated."
-  value       = "${element(concat(aws_acm_certificate_validation.wildcard.*.certificate_arn, list("")), 0)}"
+  value = element(
+    concat(
+      aws_acm_certificate_validation.wildcard.*.certificate_arn,
```

Comment:
Couple of notes here that I forgot. You don't need `element()` anymore, but can just subset with `[0]` directly. And the splat syntax going forward is going to be `[*].attribute` instead of `.*.attribute`. All in all it becomes:

```
concat(aws_acm_certificate_validation.wildcard[*].certificate_arn, [""])[0]
```